css/properties/content.json

```
@@ -48,6 +48,54 @@
             "deprecated": false
           }
         },
+        "alt_text": {
+          "__compat": {
+            "description": "Alt text",
```

The way this works is so cryptic. Maybe we can relate this to the specification a little more?

```suggestion
            "description": "Alternative text after <code>/</code>",
```